### PR TITLE
Updated DB queries

### LINKED ISSUE #124: Update/Test Database Queries based on Logged-in User
I need to update the Firestore queries to be based on the ID of the person logged in.

I also will add a test button that will log in with a test user
Email: test@user.ca
Username: testuser

### PR DESCRIPTION
**Pull request by jacobwong98**
[//]: # (These are comments and are used for reference and will not show up in the PR)

Description: 

Updated the Firestore queries with the new structure to have the Users collection contain the VibeEvent information under the logged-in user.

I will need to clean up the Database and delete all the current users that have been created as they no longer follow the new structure

[//]: # (Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change)

[//]: # (automatically closes given issue number)
Closes #124   

Proof of Concept

image

[//]: # (Attach Image of feature added or fix applied and working)
[//]: # (Please provide images of tests passing)

Checklist

[//]: # (Add an x between square brackets to mark item as done)

- [x] Follows style guidelines and is **properly commented**
- [x] Code changes generate no new warnings
- [ ] Testing (Unit and/or Intent)
- [ ] Documentation (UML, Wiki, etc.)

### CODE REVIEW

File: app/src/main/java/com/example/vybe/AddEdit/AddEditVibeEventActivity.java [hunk 1]
```diff
@@ -74,6 +77,8 @@ protected void onCreate(Bundle savedInstanceState) {
         mapFragment = (MapFragment) getSupportFragmentManager().findFragmentById(R.id.add_edit_map_fragment);
         socStnFragment = (SocialSituationFieldFragment) getSupportFragmentManager().findFragmentById(R.id.social_situation_field_fragment);
 
+        vibeEventDBPath = "Users/" + mAuth.getCurrentUser().getUid() + "/VibeEvents";
```
Comment (TheBroMoe) [inline app/src/main/java/com/example/vybe/AddEdit/AddEditVibeEventActivity.java]: At some point we should probably have a refactor issue to change string concatenations to `String.format` but this is preference for the most part

File: app/src/main/java/com/example/vybe/MyVibesActivity.java [hunk 2]
```diff
@@ -90,6 +91,8 @@ protected void onCreate(Bundle savedInstanceState) {
         socialBtn = findViewById(R.id.social_btn);
         profileBtn = findViewById(R.id.profile_btn);
 
+        vibeEventDBPath = "Users/" + mAuth.getCurrentUser().getUid() + "/VibeEvents";
+        Log.d(TAG, "onCreate: " + vibeEventDBPath);
```
Comment (TheBroMoe) [inline app/src/main/java/com/example/vybe/MyVibesActivity.java]: Do we want to keep this log for now for debugging?

File: app/src/main/java/com/example/vybe/MyVibesActivity.java [hunk 3]
```diff
@@ -175,7 +178,7 @@ public void onNothingSelected(AdapterView<?> adapterView) {}
         myVibesAdapter = new MyVibesAdapter(this, R.layout.my_vibe_item, vibeEventList);
         vibesListView.setAdapter(myVibesAdapter);
 
-        final CollectionReference collectionReference = db.collection("VibeEvent");
+        final CollectionReference collectionReference = db.collection(vibeEventDBPath);
```
Comment (TheBroMoe) [inline app/src/main/java/com/example/vybe/MyVibesActivity.java]: Why is this declared final?

File: app/src/main/java/com/example/vybe/AddEdit/AddEditVibeEventActivity.java (hunk 1 above)
Comment (jacobwong98) [reply app/src/main/java/com/example/vybe/AddEdit/AddEditVibeEventActivity.java]: Agree

File: app/src/main/java/com/example/vybe/MyVibesActivity.java (hunk 2 above)
Comment (jacobwong98) [reply app/src/main/java/com/example/vybe/MyVibesActivity.java]: I can remove the log. It was more for my testing when I was doing the work

Comment (jacobwong98) [reply app/src/main/java/com/example/vybe/MyVibesActivity.java]: Removed

File: app/src/main/java/com/example/vybe/MyVibesActivity.java (hunk 3 above)
Comment (jacobwong98) [reply app/src/main/java/com/example/vybe/MyVibesActivity.java]: Because lol. I removed it and everything works fines

**Pull request merged**